stonesoup/types/numeric.py

```diff
@@ -231,16 +233,15 @@ def log(self):
     @classmethod
     def sum(cls, values):
         """Carry out LogSumExp"""
-        log_values = [cls._log(value) for value in values]
-        if not log_values:
+        log_values = np.array([cls._log(value) for value in values])
+        if not len(log_values):
```

Comment:
Very minor comment here: Could this be changed to `if len(log_values) == 0` for the sake of readability?